src/pull-request.js

```diff
@@ -47,6 +47,12 @@ export async function run() {
     core.debug(`Pull Request: ${JSON.stringify(pullRequest)}`);
     core.debug(`Is Draft: ${JSON.stringify(isDraft)}`);
 
+    // Ignore release PRs.
+    if ( pullRequest?.head.ref && pullRequest?.head.ref.startsWith("release/") ) {
```

Comment:
Minor thing but seems we don't have spaces within parentheses elsewhere in this file so to maintain consistency, might want to remove those here
```suggestion
    if (pullRequest?.head.ref && pullRequest?.head.ref.startsWith("release/")) {
```